### DISCUSSION TITLE
Question about slave streaming server in case your main goes down

### DISCUSSION THREAD

**Discussion by Sakretsos** [AzuraCast/AzuraCast#5427] [Q&A]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #693e437 (2022-05-24 5:50)

### What happened?

Hello, 
i want to ask a question regarding a backup streaming server in case my main server fails.
Bellow i will describe my scenario.
Lets say i have a radio A on a server with autodj disabled or enabled does not matter on url = https://url1:8000 and i use this url on my website and then the server of radio A goes down and i want a slave server with Radio B setted up to connect on the url1 as backup server until my Radio A goes up again.
Is this possible?

Thanks,
Ioannis

**Comment by YosefAdPro** [comment]
Maybe work with 2 servers
https://github.com/AzuraCast/AzuraRelay

**Comment by Sakretsos** [comment]
Hello @YosefAdPro,

Thanks so much,
Also can you refer me a good company which i can rent a loadbalancer in order to make this works?

Thanks,
Ioannis
